File: QvAbu.Web/src/app/models/questions/simple-question.ts
import { Question, QuestionType } from './question';
import { SimpleAnswer } from './simple-answer';
import { Guid } from '../guid';

export class SimpleQuestion extends Question {
  answers: SimpleAnswer[];
  simpleQuestionType: SimpleQuestionType;

  constructor(id: Guid, revision: number) {
    super();
    this.id = id;

    this.type = QuestionType.simpleQuestion;
  }
}

export enum SimpleQuestionType {
  singleChoice = 0,
  multipleChoice = 1,
  trueFalse = 2
}

File: QvAbu.Data/Models/Questions/SimpleQuestion.cs
using System.Collections.Generic;

namespace QvAbu.Data.Models.Questions
{
    public class SimpleQuestion : Question
    {
        #region Properties

        public IList<SimpleAnswer> Answers { get; set; }
        public SimpleQuestionType SimpleQuestionType { get; set; }

        #endregion
    }

    public enum SimpleQuestionType
    {
        SingleChoice = 0,
        MultipleChoice = 1,
        TrueFalse = 2
    }
}
